test/check-application

```diff
@@ -787,8 +788,17 @@ class TestApplication(testlib.MachineCase):
         self.assertIn('rw', rwmnt)
         self.assertIn(rwdir[4:], rwmnt)
 
+        def get_int(n):
+            try:
+                return int(n)
+            except ValueError:
+                return 0
```

Comment:
Hmm, doesn't `b.wait()` get along with exceptions?

Reply:
Does not seem so. It was failing without this.